CHANGELOG.md

```diff
@@ -1,48 +1,49 @@
-## 2017-03-29 - Supported Release 2.1.4
+# Changelog
 
-### Summary
+All notable changes to this project will be documented in this file.
 
-Small release with bug fixes
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-### Bug Fixes
+## [Unreleased]
 
-- Unabled to start pipes server in PowerShell process on some Windows 2008r2 images ([MODULES-6927](https://tickets.puppetlabs.com/browse/MODULES-6927))
-- Minor updates to PowerShell syntax in README examples.
+### Added
 
-## 2017-12-08 - Supported Release 2.1.3
+- Metadata for supporting Windows Server 2016 ([MODULES-4271](https://tickets.puppetlabs.com/browse/MODULES-4271))
 
-### Summary
+### Fixed
 
-Small release with bug fixes.
+- Upgrade message is improved to make .NET Framework requirements clearer when running PowerShell 2.0 ([MODULES-7011](https://tickets.puppetlabs.com/browse/MODULES-7011))
+- Fix timeout handling when the user specified a timeout parameter value of `0` ([MODULES-7018](https://tickets.puppetlabs.com/browse/MODULES-7018))
```

Comment:
I'd clarify here that is assigns a sane default of 300 seconds to be clear about the change for a user.